transmit-lib/src/cmr/transmit/kms.clj

```diff
@@ -198,6 +208,14 @@
         "Completed KMS Request to %s in [%d] ms" url (- (System/currentTimeMillis) start)))
     (:body response)))
 
+(defn- uses-xml?
+  "For a given keyword-scheme, returns true if it uses an xml resource file.
+   Returns false otherwise."
+  [keyword-scheme]
+  (or
+   (= "concepts" (name keyword-scheme))
+   (= "iso-topic-categories" (name keyword-scheme))))
```

Comment:
Is there any reason you're not doing `(= :concepts keyword-scheme)` and `(= :iso-topic-categories keyword-scheme)`?